### DISCUSSION TITLE
Does anyone need Vagrant anymore?

### DISCUSSION THREAD

**Discussion by phax** [phax/phoss-smp#202] [Q&A]
Quick question to the community: 
Is anyone still using Vagrant or have you all migrated to Docker?

**Comment by ic-officient** [comment]
We are using docker. We might move to nerdctl in the future.

**Reply by phax** [reply]
Thanks for the swift response. Would that imply any change requests to the Docker image that you already know of?

Btw. I assume it is https://github.com/containerd/nerdctl

**Comment by phax** [comment]
Vagrant will be removed for the next major release
